cirq/value/periodic_value.py

```diff
@@ -76,6 +76,9 @@ def _approx_eq_(self, other: Any, atol: float) -> bool:
 
         return cirq.protocols.approx_eq(low, high, atol=atol)
 
+    def __repr__(self):
+        return 'cirq.PeriodicValue({!r}, {!r})'.format(self.value, self.period)
```

Comment:
I think value may be a symbol, and sympy.symbol unfortunately has a bad repr. There's a workaround method to deal with it.

Add a test checking this case.

Reply:
Done